public/js/applicantForm.js

```diff
@@ -15,7 +15,14 @@
     checkedCohortDates();
 
     if (document.querySelector('#chosenCourseId')) {
-        outputCohorts(data.cohorts, document.querySelector('#chosenCourseId'));
+        const chosenCourseId = document.querySelector('#chosenCourseId').dataset.selected;
+        const futureCohorts = filterFutureDates(data.cohorts);
+        if (chosenCourseId) {
+            futureCohorts.push(((data.cohorts.filter((cohort) => {
+                return cohort.id == chosenCourseId;
+            }))[0]));
```

Comment:
could this go into its own function perhaps? give it a semantic name so that its clear what its doing